module/tools/FirmwareInstaller/src/FirmwareInstaller.cpp

```diff
@@ -56,8 +56,8 @@ namespace module::tools {
 
             for (const auto& f : config["firmwares"].config) {
                 std::pair<std::string, std::string> name;
-                name.first  = utility::strutil::toUpper(f["device"].as<std::string>());
-                name.second = utility::strutil::toUpper(f["battery"].as<std::string>());
+                name.first  = utility::strutil::to_upper(f["device"].as<std::string>());
+                name.second = utility::strutil::to_pper(f["battery"].as<std::string>());
```

Comment:
Seems unrelated to this PR?

Reply:
Also that just seems like a bug

Reply:
Cant believe this one got past the build checks lol, should get fixed by tomorrow hopefully

Reply:
This is because this module isn't in any roles, so it's not being built.

Reply:
Reverted.